### ISSUE TITLE
Unable to set images bundled within a jar file

### ISSUE THREAD

**Issue by hinca** [dorkbox/SystemTray#154]
It is common to bundle and use resources within the jar file, however, it's not possible to set a tray or menu image from a path that leads to a resource within a jar.

Speficially, to get the image from `file:\C:\path\application.jar!\images\image.png` the workaround:
`systemTray.setImage(javax.imageio.ImageIO.read(getClass().getResource("/images/image.png")));` works fine, however:
`systemTray.setImage(getClass().getResource("/images/image.png").getPath());` results in the following exception although the image exists:

```
[ERROR] 2021-04-15 09:24:03.120 [JavaFX Application Thread] SystemTray:125 - Error reading image. Using error icon instead java.io.FileNotFoundException: file:\C:\path\application.jar!\images\image.png (The filename, directory name, or volume label syntax is incorrect)
	at java.io.FileInputStream.open0(Native Method) ~[?:?]
	at java.io.FileInputStream.open(FileInputStream.java:211) ~[?:?]
	at java.io.FileInputStream.<init>(FileInputStream.java:153) ~[?:?]
	at java.io.FileInputStream.<init>(FileInputStream.java:108) ~[?:?]
	at dorkbox.systemTray.util.ImageResizeUtil.resizeAndCache(ImageResizeUtil.java:118) ~[SystemTray-4.1.jar:2021-04-10T08:06:49.252271700Z]
	at dorkbox.systemTray.util.ImageResizeUtil.shouldResizeOrCache(ImageResizeUtil.java:293) ~[SystemTray-4.1.jar:2021-04-10T08:06:49.252271700Z]
	at dorkbox.systemTray.SystemTray.setImage(SystemTray.java:965) ~[SystemTray-4.1.jar:2021-04-10T08:06:49.252271700Z]
...
```

Suggestion is to use javax.imageio.ImageIO or a similar solution in ImageResizeUtil, instead of working directly with files/streams.

**Comment by dorkbox** [comment]
This makes sense. It should be super easy to add this to the next release.

**Comment by dorkbox** [comment]
If you pass in a stream, it will properly load the resource from it (instead of passing in a path, and expecting it to parse correctly).

For example: `systemTray.setImage(getClass().getResource("/images/image.png"));` will load the image as you expect. It should also work now via passing in the path to a jar resource, however this is not really the best way to do this.

Fixed: a3db2149
